### ISSUE TITLE
linstor --curl snapshot list. IndexError: list index out of range.

### ISSUE THREAD

**Issue by kvaps** [LINBIT/linstor-client#34]
```bash
# linstor --curl snapshot list                                        
curl -X GET http://localhost:3370/v1/resource-definitions?with_volume_definitions=true
Traceback (most recent call last):
  File "/usr/bin/linstor", line 24, in <module>
    linstor_client_main.main()
  File "/usr/lib/python3/dist-packages/linstor_client_main.py", line 668, in main
    LinStorCLI().run()
  File "/usr/lib/python3/dist-packages/linstor_client_main.py", line 633, in run
    rc = self.parse_and_execute(sys.argv[1:])
  File "/usr/lib/python3/dist-packages/linstor_client_main.py", line 347, in parse_and_execute
    rc = args.func(args)
  File "/usr/lib/python3/dist-packages/linstor_client/commands/snapshot_cmds.py", line 259, in list
    lstmsg = self._linstor.snapshot_dfn_list()
  File "/usr/lib/python3.6/dist-packages/linstor/linstorapi.py", line 2546, in snapshot_dfn_list
    rsc_dfns = self.resource_dfn_list()[0]
IndexError: list index out of range
```

**Comment by rp-** [comment]
This will be fixed, once we have a dedicated snapshot list view, than it will be just a single rest call to get all the snapshots.

**Comment by rp-** [comment]
should be fixed with release 1.7.0 and 1.1.0
